ci/autoscaler/scripts/deploy-autoscaler.sh

```diff
@@ -80,21 +80,53 @@ function deploy () {
   ${script_dir}/silence_prometheus_alert.sh "BOSHJobUnhealthy"
   set -e
 
+  # Set the local tmp_dir depending on if we run on github-actions or not, see:
+  # https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+  local tmp_dir
+  local perform_as_gh_action
+  perform_as_gh_action="${GITHUB_ACTIONS:-false}"
+  if "${perform_as_gh_action}" != 'false'
+  then
+    tmp_dir="${RUNNER_TEMP}"
+  else # local system
+    tmp_dir="$(pwd)/dev_releases"
+    mkdir -p "${tmp_dir}"
+  fi
+
+  local tmp_manifest_file
+  # on MacOS mktemp does not know the --tmpdir option
+  tmp_manifest_file="$(mktemp "${tmp_dir}/${deployment_name}.bosh-manifest.yaml.XXX")"
```

Comment:
Would it maybe be useful for the files to end with `.yaml` so that editor show it with syntax highlighting?

```suggestion
  tmp_manifest_file="$(mktemp "${tmp_dir}/${deployment_name}.bosh-manifest.XXX.yaml")"
```